src/messages.js

```diff
@@ -30,4 +30,10 @@ export default {
   'field.places_common.termStatus.name': 'Admin status',
   'vocab.place.local.name': 'Default',
   'vocab.place.local.collectionName': 'Default places',
+
+  // Organization
+  'vocab.organization.local.name': 'Organization',
+  'vocab.organization.local.fullName': 'Organizations',
```

Comment:
Should be `collectionName`, not `fullName` -- be sure to verify your config by checking in the UI. collectionName appears in the title of search results.